ui/src/common/SongDatagrid.js

```diff
@@ -49,12 +51,57 @@ const useStyles = makeStyles({
   },
 })
 
+const EditionRow = forwardRef(
+  ({ record, onClick, colSpan, contextAlwaysVisible}, ref) => {
+    const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'))
+    const classes = useStyles({ isDesktop })
+    const translate = useTranslate()
+    const handlePlaySubset = (releaseDate) => () => {
+      onClick(releaseDate)
+    }
+
+    let editionTitle = []
+    if (record.releaseDate) {
+      editionTitle.push(translate('resources.album.fields.released'))
+      editionTitle.push(FormatFullDate(record.releaseDate))
+      if (record.catalogNum) {
+        editionTitle.push("· Cat #")
+        editionTitle.push(record.catalogNum)
+      }
+    }
+
+    return (
+      <TableRow
+        hover
+        ref={ref}
+        onClick={handlePlaySubset(record.releaseDate)}
+        className={classes.row}
+      >
+        <TableCell colSpan={colSpan}>
```

Comment:
This has the potential to overflow if the category number is too long; maybe add either some word wrap control? (See your "Seventh Son of a Seventh Son" example)